### ISSUE TITLE
Pytorch Autograd no_grad equivalent in Keras

### ISSUE THREAD

**Issue by tonmoyborah** [keras-team/keras#13313]
Hi, I am not sure if this is a bug/ feature, more of a general issue. I recently came to know of a feature in `pytorch` Autograd where we can pass `no_grad` as described [here](https://pytorch.org/docs/stable/autograd.html#torch.autograd.no_grad). I read somewhere that pytorch calculates gradients even during inference and ends up consuming more than 1GB GPU RAM due to it. This feature allows users to stop this and save space (I don't use pytorch so not 100pc sure). I am looking around Keras to find if Keras does this too? And is there anything we can do to stop this? (Yet to find something). Any pointers would be appreciated.
Thanks!

**Comment by fchollet** [comment]
You can use `tf.stop_gradients`: https://www.tensorflow.org/api_docs/python/tf/stop_gradient

**Comment by fchollet** [comment]
Note that Keras does not compute gradients when using `predict` (or just calling the model via `__call__`) so it doesn't have this specific issue.
